src/renderer/components/app.scss

```diff
@@ -105,7 +105,7 @@ h4 {
 h5 {
   @extend h4;
   padding: $padding / 2 0;
-  font-size: 14px;
+  font-size: 16px;
```

Comment:
Why do we want h5 to be the same as h4?

Reply:
Make sense. Made it bigger `18px` than h5. We don't use any of `h4` currently, but it's better for general purposes.